src/main/java/io/kokuwa/maven/helm/github/Github.java

```diff
@@ -92,7 +100,8 @@ private Optional<ReleaseCache> readCache() {
 
 		try {
 			ReleaseCache cache = mapper.readValue(file.toFile(), ReleaseCache.class);
-			log.debug("Github cache found at " + file);
+			cache.setTimestamp(Files.getLastModifiedTime(file).toInstant());
```

Comment:
I would like to set this date as time of creation a cache during cache create.

Somebody can store cache file in shared resources, which can be copied between nodes, so file modification time can change.

Reply:
Good point. I will include the timestamp in json file.

Reply:
Rafactored, can you check again?